torch/lib/c10d/test/ProcessGroupMPITest.cpp

```diff
@@ -36,13 +41,14 @@ void testAllreduce(int iter = 1000) {
 
   std::vector<c10::intrusive_ptr<::c10d::ProcessGroup::Work>> works;
   for (auto& tensors : allTensors) {
-    // Kick off work
```

Comment:
I thought it's misleading since it = puts work in queue and doesn't kick it off. the line is 
work = pg->allreduce(tensors);
is self-explanatory.

Changed comments here and in the rest of module